include/stdexec/execution.hpp

```diff
@@ -1243,6 +1244,30 @@ namespace stdexec {
   /////////////////////////////////////////////////////////////////////////////
   // __connect_awaitable_
   namespace __connect_awaitable_ {
+    inline constexpr auto __co_call =
+      []<class _Fun, class... _Ts>(_Fun __fun, _Ts&&... __as) noexcept {
+        auto __fn = [&, __fun]() noexcept {
+          __fun((_Ts&&) __as...);
+        };
+
+        struct __awaitable {
+          decltype(__fn) __fn_;
+
+          static constexpr bool await_ready() noexcept {
+            return false;
+          }
+
+          void await_suspend(__coro::coroutine_handle<>) noexcept {
+            __fn_();
+          }
+
+          void await_resume() noexcept {
+          }
```

Comment:
Annotate with `[[noreturn]]` and put `std::terminate()` in here (or `std::unreachable()`).